### PR TITLE
Fixed vale errors and added missing descriptions

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

_What did you add/update/change?_

## Type of suggestion

* [x] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [x] Other

## Product & version (if relevant)

Engage v13

## Deadline (if relevant)

Anytime

### CODE REVIEW

File: 13/umbraco-engage/marketers-and-editors/personalization/implicit-and-explicit-personalization/campaign-scoring.md [hunk 1]
```diff
@@ -56,19 +58,19 @@ You can add a new campaign group by clicking "**Add new group**". This will open
...
 
 An example of this tab could look like this:
 
 <figure><img src="../../../.gitbook/assets/image (8) (1) (1).png" alt="Unscored campaigns."><figcaption><p>Unscored campaigns.</p></figcaption></figure>
 
-So here you see that in the last row, for example, the **utm\_source** is "**Activate account**", the **utm\_medium** is "**email**" and the **utm\_content** is "**click here to activate your account**". The URL that the visitor used to get to the website looked something like `https://<url>?utm_source=activate account&utm_medium=email&utm_content=click here to activate your account.`
+In the last row, for example, the **utm_source** is "**Activate account**", **utm_medium** is "**email**" and **utm_content** is **click here to activate your account**. The URL that the visitor used to get to the website looked something like `https://<url>?utm_source=activate account&utm_medium=email&utm_content=click here to activate your account.`
```
Comment (sofietoft) [inline 13/umbraco-engage/marketers-and-editors/personalization/implicit-and-explicit-personalization/campaign-scoring.md:73]:
```suggestion
In the last row, for example, the `utm_source` is "**Activate account**", `utm_medium` is "**email**" and `utm_content` is **click here to activate your account**. The URL that the visitor used to get to the website looked something like `https://<url>?utm_source=activate account&utm_medium=email&utm_content=click here to activate your account.`
```

File: 13/umbraco-engage/security-and-privacy/gdpr/README.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+---
+description: >-
+  Read how Umbraco Engage is designed with GDPR compliance in mind, ensuring data privacy through first-party cookies, data anonymization, and retention controls
```
Comment (sofietoft) [inline 13/umbraco-engage/security-and-privacy/gdpr/README.md:3]:
```suggestion
  Read how Umbraco Engage is designed with GDPR compliance in mind, ensuring data privacy through first-party cookies, data anonymization, and retention controls.
```

File: 13/umbraco-engage/security-and-privacy/how-it-works.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
+---
+description: >-
+  Explore how Umbraco Engage collects browsing behavior data upon cookie acceptance to enable analytics, A/B testing, content personalization, and user profiling, all stored locally
```
Comment (sofietoft) [inline 13/umbraco-engage/security-and-privacy/how-it-works.md:3]:
```suggestion
  Explore how Umbraco Engage collects browsing behavior data upon cookie acceptance to enable analytics, A/B testing, content personalization, and user profiling, all stored locally.
```

File: 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md [hunk 4]
```diff
@@ -27,10 +27,10 @@ The first step to setting up an A/B test on your website is to define a solid pl
 This tutorial does not cover defining a plan or what makes a project valid for testing. However, find some general recommendations below.
 
 * **Set a Clear Objective:** Start by defining a specific goal for your A/B test. Knowing what you aim to improve or measure allows you to design the test accordingly. For example, do you want more clicks on a button, a higher conversion rate, or something else?
-* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the ROAR: Risk, Optimization, Automation, Re-think model to determine whether your website is eligible.
+* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the Risk, Optimization, Automation, Re-think (ROAR) model to determine whether your website is eligible.
```
Comment (sofietoft) [inline 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md:30]: Maybe this will be accepted if the commas are left out?

**Review by sofietoft — CHANGES_REQUESTED** [review]
I've added some suggestions and comments.

For the warnings on the lists, where it complains about how each item should start with a capital letter - please update that to use inline code, as that also make sense, for the parameter names 💪

File: 13/umbraco-engage/developers/settings/custom-goals-scoring.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ The method automatically determines the current page view, linking the goal to a
 
 To trigger a goal outside of an HTTP request, use the overload of **TriggerGoal** that takes the GUID of the pageview.
 
-Retrieve this pageview GUID during the original request using the **Umbraco.Engage.Business.Analytics.Common.IPageviewGuidManager**. You will need to store this pageview GUID for later use when invoking:
+Retrieve the pageview GUID in the original request using **Umbraco.Engage.Business.Analytics.Common.IPageviewGuidManager**. You will need to store this pageview GUID for later use when invoking:
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 13/umbraco-engage/developers/settings/custom-goals-scoring.md:48]: Yep. even I cannot figure out what it is complaining about 🙈

File: 13/umbraco-engage/getting-started/for-developers/infrastructure-sizing.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ The following presents recommendations based on the environment and the number o
 ## Non-cloud
 
 * **100.000 page views per month or less:** Database: CPU 2, 4-8 GB RAM, 50GB disk
-* **500.000 page views per month or less:** Database: CPU 4, 8-16 GB RAM, 100GB SSD disk
+* **500.000 page views per month or less:** Database: CPU 4, 8-16 GB RAM, 100GB Solid State Drive (SSD) disk
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 13/umbraco-engage/getting-started/for-developers/infrastructure-sizing.md:14]: yup, makes sense.. I've added it to the acronym list 💪

**Comment by eshanrnh** [comment]
Thanks @sofietoft for your review. I've incorporated your suggestions/comments 💪

File: 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md [hunk 7]
```diff
@@ -27,7 +27,7 @@ The first step to setting up an A/B test on your website is to define a solid pl
 This tutorial does not cover defining a plan or what makes a project valid for testing. However, find some general recommendations below.
 
 * **Set a Clear Objective:** Start by defining a specific goal for your A/B test. Knowing what you aim to improve or measure allows you to design the test accordingly. For example, do you want more clicks on a button, a higher conversion rate, or something else?
-* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the Risk, Optimization, Automation, Re-think (ROAR) model to determine whether your website is eligible.
+* **Ensure a valid amount of data:** A significant amount of traffic and conversions is important to produce reliable results from A/B tests. Use a tool like the Risk Optimization Automation Re-think (ROAR) model to determine whether your website is eligible.
```
Comment (sofietoft) [inline 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md:30]: Hm! This one it still complaining 😅 
Perhaps try without the hyphen in "Rethink"? 🤔

**Review by sofietoft — COMMENTED** [review]
Looks great! 💪 

Just a final suggestion, related to the ROAR acronym it's still complaining about 😅

File: 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md (hunk 7 above)
Comment (eshanrnh) [reply 13/umbraco-engage/tutorials/set-up-your-first-ab-test.md]: Yup, that works 🤩🌟

**Comment by eshanrnh** [comment]
The error is gone now 🤩

**Pull request merged**